penguin/plugins/discovery.py

```
@@ -192,6 +192,13 @@ def _discover_from_entry_points(self) -> None:
                     
         except Exception as e:
             logger.error(f"Error discovering entry point plugins: {e}")
+
+    def _iter_entry_points(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_iter_entry_points`

(ANN202)